client/ayon_houdini/plugins/publish/increment_current_file.py

```diff
@@ -42,12 +42,13 @@ def process(self, context):
                 "submission to deadline failed."
             )
 
-        # Filename must not have changed since collecting
+        # Filename must not have changed since extraction.
```

Comment:
I understand why you changed the comment - it's because there's a Save current scene extractor that ALSO happens to check something similar. Nonetheless, it checks here against the "collected" value. It's technically irrelevant whether the other extractor ran or not. So to me, the original comment was actually more correct.

```suggestion
        # Filename must not have changed since collecting.
```

Reply:
https://github.com/ynput/ayon-houdini/pull/193/commits/78e15a5ed327a27eabacc3dfa688d3d39c6e1557